nemo/collections/tts/torch/data.py

```diff
@@ -393,6 +395,8 @@ def __getitem__(self, index):
         # Let's keep audio name and all internal directories in rel_audio_path_as_text_id to avoid any collisions
         rel_audio_path = Path(sample["audio_filepath"]).relative_to(self.base_data_dir).with_suffix("")
         rel_audio_path_as_text_id = str(rel_audio_path).replace("/", "_")
+        if sample["is_phoneme"] == 1:
```

Comment:
nit: `is_phoneme` is a binary variable. Maybe we could specify it as true or false. no need to change for r1.10. I can adjust it in r1.11.